dev-tools/generate_notice.py

```diff
@@ -102,6 +134,11 @@ def create_notice(filename, beat, copyright, vendor_dirs):
 
         if 'vendor' in dirs:
             vendor_dirs.append(os.path.join(root, 'vendor'))
+            dirs.remove('vendor')   # don't walk down into sub-vendors
```

Comment:
It results in duplicates, because when we look for LICENSE files, we walk down at any level, thus catching nested vendored deps.